inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -691,7 +717,7 @@ void GNAModelSerial::ImportInputs(std::istream &is,
     dataMap.clear();
 
     for (auto inputIndex = 0; inputIndex < modelHeader.nInputs; inputIndex++) {
-        std::string name = "input" + std::to_string(inputIndex);
+        std::string name = inputNames.at(inputIndex);
```

Comment:
`std::vector::at` return by reference to constant value so you may use `const std::string&` or `const auto&` here.
https://en.cppreference.com/w/cpp/container/vector/at